### PR TITLE
Question 3

### PR DESCRIPTION
**Pull request by taruns52**
Please check and let me know if there is any possible improvement.

### CODE REVIEW

File: Question-3.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+print('\n\nword_list', word_list, '\n\n')
+
+
+word_tree = {}
+for lst in word_list:
+    for word in lst:        
+        word_tree[word] = word_tree.get(word,0) + 1
+
+print('Number of time each word appears: \n')
+print ('word_tree:', word_tree )
```
Comment (himanshu-suthar-tagline) [inline Question-3.py:42]: You can use collections.Counter here as well.
like `dict(collections.Counter(word_list))`

File: Question-1.py [hunk 2]
```diff
@@ -22,4 +22,64 @@
...
+  
+    print ('Number of duplicates in arr',numbers,'is',count,'with duplicate numbers',dups,'\n')
+
+
+# E. Display list without duplicate numbers
+elif choice.lower() == 'e':
+    print('List without duplicate numbers:',list(set(numbers)) )
+
+else:
+    print('\nInvalid Input!!!\n ')
```
Comment (himanshu-suthar-tagline) [inline Question-1.py:85]: Why this first question comes in this branch?

Comment (himanshu-suthar-tagline) [reply Question-1.py:85]: Please always create branch from main branch and not from subbranches.

Comment (taruns52) [reply Question-1.py:85]: > Please always create branch from main branch and not from subbranches.

Okay sir i will take care from now on

File: Question-3.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import collections
+
+
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding",
+              "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen."]
+
+print('\nsentences', sentences, end='\n')
+
+ 
+word_list=[ sentence.split(' ') for sentence in sentences]
```
Comment (himanshu-suthar-tagline) [inline Question-3.py]: Maybe a better way is to join the sentences by empty space `" "` and then split that result by empty list.
```
sentences = " ".join(sentences)
word_list = sentences.split(" ")
```

then you don't need to run two for loops below.

**Comment by himanshu-suthar-tagline** [comment]
Merge this.

**Pull request merged**